### PR TITLE
Fixed some consistency issues in the documentation

### PR DESCRIPTION
**Pull request by vazaha-nl**
I have been busy with parsing the markdown docs in order to get something resembling an api spec, and I found some inconsistencies and other issues, which could not be fixed in my parser code. Here's what I found so far.

### CODE REVIEW

File: content/en/entities/StatusEdit.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ aliases: [
 ### `account` {#account}
 
 **Description:** The account that published this revision.\
-**Type:** Account\
+**Type:** [Account]\
```
Comment (trwnh) [inline content/en/entities/StatusEdit.md:81]: `[Account]({{<relref "entities/Account">}})` perhaps?

**Review by trwnh — CHANGES_REQUESTED** [review]

File: content/en/entities/StatusEdit.md (hunk 1 above)
Comment (vazaha-nl) [reply content/en/entities/StatusEdit.md]: yes, you are right, fixed.

**Review by trwnh — APPROVED** [review]

**Comment by vazaha-nl** [comment]
Please let me know how I can help to get this merged. Thanks!

**Comment by vazaha-nl** [comment]
Hello? Anyone?

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**